Composer/packages/client/src/pages/design/exportSkillModal/content/SelectProfile.tsx

```diff
@@ -24,6 +27,16 @@ const styles = {
   `,
 };
 
+const actionButton = {
+  root: {
+    fontSize: 14,
+    fontWeight: FontWeights.regular,
+    color: SharedColors.cyanBlue10,
```

Comment:
why are you overwriting default color of the ActionButton?

Reply:
use `messagebar`